src/bodywork/cli/secrets.py

```diff
@@ -140,12 +140,21 @@ def display_secrets(
     if secret_name and not group:
         print_warn("Please specify which secrets group the secret belongs to.")
         return None
-    elif secret_name and group:
+    if secret_name:
         try:
-            complete_secret_name = _create_complete_secret_name(group, secret_name)
-            print_dict(secrets[complete_secret_name].data, complete_secret_name)
+            complete_secret_name = _create_complete_secret_name(
+                str(group), str(secret_name)
+            )
+            print_dict(
+                secrets[complete_secret_name].data,
+                _create_table_name(str(secret_name), str(group)),
```

Comment:
Really shouldn't be repeat casting type just to appease mypy. I know it's frustrating, but...

Reply:
Completely agree, can we disable MyPy for this sh*t because all i'm getting out of it, is being forced to write sub-optimal code due to MyPy's shortcomings.

Reply:
The above change from the previous comment should fix this, as it's explicit enough for mypy to figure-out what's going wrong.

As I said before, let's leave this for now and come-up with a Mypy strategy (or replacement) after v3 is done. It may be, that we end-up configuring it just for function signatures or something.